.github/locks/osx-64_test_lab3.6_py3.11.conda.lock

```diff
@@ -6,7 +6,7 @@
 #   - firefox =102
 #   - geckodriver
 #   - importnb
-#   - ipywidgets >=8.0.4,<9
+#   - ipywidgets >=8.0.0
```

Comment:
is this `<9` no longer valid?  I noticed it was still used the `pyproject.toml` file